service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
-. Login to your OpenShift cluster using `oc login` command line tool
+
+. If you are not already logged in to your OpenShift cluster, log in as a user (such as a cluster administrator) that has privileges to create a new project in the cluster. For example:
++
+[source, subs="+quotes"]
+----
+$ oc login -u system:admin -p __<password>__ --server=__<host:port>__
+----
+
+. Ensure that the current OpenShift project is the one created in the previous step of this tutorial. For example:
 +
 [source]
 ----
-oc login
+$ oc project rhoas-quarkus
 ----
-. Create new namespace `rhoas-quarkus` that will be used for this guide
+
+. Connect a Kafka instance in {product-long} to the current project in your OpenShift cluster.
 +
 [source]
 ----
-oc new-project rhoas-quarkus
+$ rhoas cluster connect --ignore-context
 ----
 +
-. Login to rhoas CLI
+You are prompted to specify the Kafka instance that you want to connect to OpenShift.
+
+. Type the name of the Kafka instance that you want to connect to OpenShift. Press *Enter*.
++
+You should see output like the following:
 +
 [source]
 ----
-rhoas login
+Connection Details:
+
+Apache Kafka instance:  my-kafka-instance
+Kubernetes Namespace:   rhoas-quarkus
+Service Account Secret: rh-cloud-services-service-account
 ----
+
+. Verify the connection details shown by the CLI. When you are ready to continue, type `y`. Then, press *Enter*.
 +
-. On the command line, enter  `rhoas cluster status` command and execute check if CLI has proper connection to cluster
+You are prompted to provide an access token. The RHOAS Operator requires this token to make a connection to your Kafka instance.
+
+. In your web browser, open the link:https://cloud.redhat.com/openshift/token[OpenShift Cluster Manager API Token] page. Copy the access token shown.
+
+. In your terminal window, right-click and select *Paste*. Press *Enter*.
++
+The Operator uses the token to create a `KafkaConnection` resource on your OpenShift cluster. When this process is complete, you should see lines like the following:
 +
 [source]
 ----
-rhoas cluster status
+KafkaConnection resource "my-kafka-instance" has been created
+Waiting for status from KafkaConnection resource.
+Created KafkaConnection can be injected into your application.
+...
+KafkaConnection successfully installed on your cluster.
 ----
 
-ifdef::qs[]
-.Verification
-* Is RHOAS Operator installation finished successfully
-* Please inspect output of the command and check if RHOAS Operator is installed.
-* Output from the command contains: "RHOAS Operator: Installed"
-* Namespace is `rhoas-quarkus`
-endif::[]
+. Verify that the Operator successfully created the connection.
++
+[source]
+----
+$ oc get KafkaConnection
 
-[id="proc-connecting-kafka_{context}"]
-== Connecting Apache Kafka Service to your cluster
+NAME   		         AGE
+my-kafka-instance        2m35s
+----
++
+As shown in the output, the Operator creates a `KafkaConnection` resource that matches the name of your Kafka instance.
```

Comment:
Verification should always have `?` - I think :)

Something like:

Is your `KafkaConnection` resource that matching the name of your Kafka instance.

Reply:
Discussed in person with @wtrocki. This content is not QS content, and therefore does not include a Verification section.